### PR TITLE
【課題解決Wiki】ビルド時に"Index was out of range. Must be non-negative and less than the size of the collection."が表示される

### PR DESCRIPTION
**Pull request by ra9g16**
### **PR Type**
Documentation

___

### **Description**
- 新しいFAQドキュメントを追加し、ビルド時に発生する"Index was out of range"エラーについて説明。
- 英語と日本語の両方で現象、発生環境、解決の試みを詳述。
- チェンジログを更新し、新しいFAQエントリを追加。
- mkdocsのナビゲーションを更新し、新しいFAQエントリを含める。

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>UV2MeshBuildError.en.md</strong><dd><code>Add FAQ for "Index out of range" build error in English</code>&nbsp; &nbsp; </dd></summary>
<hr>

docs/FAQ/UV2MeshBuildError.en.md

<li>Added a new FAQ document explaining the "Index was out of range" <br>error.<br> <li> Described the phenomenon, environment, and resolution attempts.<br> <li> Provided a conclusion and additional insights.<br>

</details>

  </td>
  <td><a href="https://github.com/VRHIKKY/VketCloudSDK_Documents/pull/554/files#diff-e1c18d68d18fd3da3debd082f31fd9c117eb84055c12c1dc1457a5db25dc6bd9">+38/-0</a>&nbsp; &nbsp; </td>

</tr>

<tr>
  <td>
    <details>
      <summary><strong>UV2MeshBuildError.ja.md</strong><dd><code>Add FAQ for "Index out of range" build error in Japanese</code>&nbsp; </dd></summary>
<hr>

docs/FAQ/UV2MeshBuildError.ja.md

<li>Added a new FAQ document in Japanese for the "Index was out of range" <br>error.<br> <li> Included details on the phenomenon, environment, and resolution <br>attempts.<br> <li> Offered a conclusion and additional insights.<br>

</details>

  </td>
  <td><a href="https://github.com/VRHIKKY/VketCloudSDK_Documents/pull/554/files#diff-ce0cdcad3cea09d52983b1b227d9c12bf2ff1920c941df1514992269a623b224">+38/-0</a>&nbsp; &nbsp; </td>

</tr>

<tr>
  <td>
    <details>
      <summary><strong>changelog-14.2.en.md</strong><dd><code>Update changelog with new FAQ entry in English</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

docs/changelog/changelog-14.2.en.md

- Updated changelog to include the new FAQ for build error.

</details>

  </td>
  <td><a href="https://github.com/VRHIKKY/VketCloudSDK_Documents/pull/554/files#diff-2f3baf40cb41ed78decf8d39000daef3a13e7b63f50de72c9d8a685e1356d6bb">+1/-0</a>&nbsp; &nbsp; &nbsp; </td>

</tr>

<tr>
  <td>
    <details>
      <summary><strong>changelog-14.2.ja.md</strong><dd><code>Update changelog with new FAQ entry in Japanese</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

docs/changelog/changelog-14.2.ja.md

<li>Updated changelog to include the new FAQ for build error in Japanese.<br>

</details>

  </td>
  <td><a href="https://github.com/VRHIKKY/VketCloudSDK_Documents/pull/554/files#diff-a8f7442620adccd75c20db969e01eb0df04d1ae71c0bdda20f51ce3726dd1a5b">+1/-0</a>&nbsp; &nbsp; &nbsp; </td>

</tr>
</table></td></tr><tr><td><strong>Configuration changes</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>mkdocs.yml</strong><dd><code>Update mkdocs navigation with new FAQ entry</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

mkdocs.yml

<li>Added new FAQ entry to the documentation navigation.<br> <li> Updated both English and Japanese sections.<br>

</details>

  </td>
  <td><a href="https://github.com/VRHIKKY/VketCloudSDK_Documents/pull/554/files#diff-98d0f806abc9af24e6a7c545d3d77e8f9ad57643e27211d7a7b896113e420ed2">+2/-0</a>&nbsp; &nbsp; &nbsp; </td>

</tr>
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**: Comment `/help "your question"` on any pull request to receive relevant information

### CODE REVIEW

**Review by Kondo-Totori — COMMENTED** [review]
uv2を含まないメッシュであっても、SkinnedMesh Rendererを用いることでビルドが通り、VketCloudに反映可能。
↓
uv2を含まないメッシュであっても、SkinnedMesh Rendererを用いることでビルドが通り、Vket Cloudに反映可能。

表記揺れの対策をお願いいたします。

File: docs/FAQ/UV2MeshBuildError.ja.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+| ![正常にビルドできたオブジェクト](img/UV2MeshBuildError_2.jpg) | ![エラーが発生したオブジェクト](img/UV2MeshBuildError_3.jpg) |
+
+見比べたところ、uv2の表示がない事に気が付いた。
+エラーの原因はuv2がないのにMesh Rendererを使用しようとしていたことだった。
+
+## 結論
+uv2のないメッシュをMesh Rendererで扱おうとするとビルドエラーになる。
+
+uv2を含むようにモデリング班にメッシュの作り直し依頼を行うか、
+SkinnedMesh Rendererを使用するか、
```
Comment (Negi-T) [inline docs/FAQ/UV2MeshBuildError.ja.md:31]: SkinnedMesh --> Skinned Mesh

Comment (ra9g16) [reply docs/FAQ/UV2MeshBuildError.ja.md]: https://github.com/VRHIKKY/VketCloudSDK_Documents/pull/554/commits/0f46fc8164aa626849c0479c319b67a4412bc65c 
こちら修正済みです！

**Comment by ra9g16** [comment]
> uv2を含まないメッシュであっても、SkinnedMesh Rendererを用いることでビルドが通り、VketCloudに反映可能。 ↓ uv2を含まないメッシュであっても、SkinnedMesh Rendererを用いることでビルドが通り、Vket Cloudに反映可能。
> 
> 表記揺れの対策をお願いいたします。

以下Commitで修正済みです！
https://github.com/VRHIKKY/VketCloudSDK_Documents/pull/554/commits/0f46fc8164aa626849c0479c319b67a4412bc65c

**Review by Kondo-Totori — APPROVED** [review]
LGTM!

**Review by Negi-T — APPROVED** [review]
LGTM!

**Pull request merged**